File: May 3, 2026.py
def get_greeting(s):

    hours, minutes = map(int, s.split(":"))
    total_minutes = hours * 60 + minutes

    if (total_minutes >= 5 * 60) and (total_minutes < 12 * 60):
        return "Good morning"
    elif (total_minutes >= 12 * 60) and (total_minutes < 18 * 60):
        return "Good afternoon"
    elif (total_minutes >= 18 * 60) and (total_minutes < 22 * 60):
        return "Good evening"

    return "Good night"

File: May 3, 2026.js
function getGreeting(time) {

  const [hours, minutes] = time.split(":").map(Number);
  const totalMinutes = hours * 60 + minutes;

  if ((totalMinutes >= 5 * 60) && (totalMinutes < 12 * 60)) {
    return "Good morning";
  } else if ((totalMinutes >= 12 * 60) && (totalMinutes < 18 * 60)) {
    return "Good afternoon";
  } else if ((totalMinutes >= 18 * 60) && (totalMinutes < 22 * 60)) {
    return "Good evening";
  }

  return "Good night";
}